packages/data/README.md

```diff
@@ -516,44 +516,7 @@ _Returns_
 
 <a name="withSelect" href="#withSelect">#</a> **withSelect**
 
-Higher-order component used to inject state-derived props using registered
-selectors.
-
-_Usage_
-
-```js
-function PriceDisplay( { price, currency } ) {
-	return new Intl.NumberFormat( 'en-US', {
-		style: 'currency',
-		currency,
-	} ).format( price );
-}
-
-const { withSelect } = wp.data;
-
-const HammerPriceDisplay = withSelect( ( select, ownProps ) => {
-	const { getPrice } = select( 'my-shop' );
-	const { currency } = ownProps;
-
-	return {
-		price: getPrice( 'hammer', currency ),
-	};
-} )( PriceDisplay );
-
-// Rendered in the application:
-//
-//  <HammerPriceDisplay currency="USD" />
-```
-
-In the above example, when `HammerPriceDisplay` is rendered into an application, it will pass the price into the underlying `PriceDisplay` component and update automatically if the price of a hammer ever changes in the store.
-
-_Parameters_
-
--   _mapSelectToProps_ `Function`: Function called on every state change, expected to return object of props to merge with the component's own props.
-
-_Returns_
-
--   `Component`: Enhanced component with merged state data props.
```

Comment:
This documentation will get replace once I add inline docs to the new `withSelect`.  I just want to gather initial feedback on approach before doing so.

Reply:
I guess this needs an update too.